Onboarding/TechicalOverview.md

```diff
@@ -0,0 +1,31 @@
+# CanDIG Technical Onboarding
+&nbsp;
```

Comment:
Why do I see `&nbsp;`?

Reply:
I'm actually using that add an empty line. Couldn't seem to find a better way

Reply:
Better to remove this rather than have that entity there. Newline would do instead of that.

Reply:
Newline only seems to work if it is succeeded by non-header text. I'll take out the nbsp and leave the doc without any special spacing for now.